front/app/containers/Admin/projects/project/participationContext/components/inputs/VotingInputs/index.tsx

```diff
@@ -123,9 +93,20 @@ export default ({
           }
         />
       )}
-      {/* Render any voting method specific inputs from configuration */}
-      {votingMethodConfig?.getVotingMethodInputs &&
-        votingMethodConfig.getVotingMethodInputs(props)}
```

Comment:
Same as other comment I had - why did we remove the new configuration and use <BudgetingInputs> instead? When we use the configuration, we only need this one line to add in method-specific fields. But if we do it this way, we'll end up adding in
separate <MultipleVotesInputs>, <SingleVoteInputs>, <RankedVoteInputs> etc. to this file for each new voting method right?